### ISSUE TITLE
example in document not work in playground

### ISSUE THREAD

**Issue by bronze1man** [vlang/v#3046]
**V version:** V playground
**OS:** V playground
**What did you do?**
```v
fn main() {
	mut m := map[string]int // Only maps with string keys are allowed for now 
	m['one'] = 1
	m['two'] = 2
	println(m['one']) // "1" 
	println(m['bad_key']) // "0" 
	println('bad_key' in m) // Use `in` to detect whether such key exists
	m.delete('two')

	numbers := {
		'one': 1,
		'two': 2,
	}
	println(numbers['one'])
}
```
with error message 
```
syntax error: unexpected `}`, expecting `STR`
   12| 	'two': 2,
```

**Comment by lutherwenxu** [comment]
This is actually a problem with the documentation. The correct code would be:
```go
	numbers := {
		'one': 1,
		'two': 2
	}
```

**Comment by bronze1man** [comment]
Please support `,` in the last element of the map.
It is very easy to write wrong code in json too. 
```json
{"one": 1,"two": 2,}
```

**Comment by lutherwenxu** [comment]
> Please support `,` in the last element of the map.
> It is very easy to write wrong code in json too.
> 
> ```json
> {"one": 1,"two": 2,}
> ```

From what I know, this definitely contradicts the one-way philosophy. You could try asking in the Discord server tho.

This issue is kept open to track the out-of-date documentation.

**Comment by noonat** [comment]
Looks like the documentation was fixed by #3454.

**Comment by lutherwenxu** [comment]
It indeed has.
